python/pyarrow/tests/test_parquet.py

```diff
@@ -1523,6 +1523,30 @@ def test_invalid_pred_op(tempdir):
                           ])
 
 
+@pytest.mark.pandas
+def test_filters_read_table(tempdir):
+    # test that filters keyword is passed through in read_table
+    fs = LocalFileSystem.get_instance()
+    base_path = tempdir
+
+    integer_keys = [0, 1, 2, 3, 4]
+    partition_spec = [
+        ['integers', integer_keys],
+    ]
+    N = 5
+
+    df = pd.DataFrame({
+        'index': np.arange(N),
+        'integers': np.array(integer_keys, dtype='i4'),
+    }, columns=['index', 'integers'])
+
+    _generate_partition_directories(fs, base_path, partition_spec, df)
+
+    table = pq.read_table(
```

Comment:
Can you also add a test with `[[('integers', '<', 3)]]`? I consider the `[[…]]` to be better for end users as it supports the full scope of all possible queries.

Reply:
Added one (although I think it is not that essential here, as I am only testing the filter argument is correctly passed through)